exchanges/newexchange.md

```diff
@@ -0,0 +1,1061 @@
+| Exchange | REST API | Streaming API | FIX API |
+|----------|------|-----------|-----|
+| Alphapoint | Yes  | Yes        | NA  |
+| Binance| Yes  | Yes        | NA  |
+| Bitfinex | Yes  | Yes        | NA  |
+| Bitflyer | Yes  | No      | NA  |
+| Bithumb | Yes  | NA       | NA  |
+| BitMEX | Yes | Yes | NA |
+| Bitstamp | Yes  | Yes       | No  |
+| Bittrex | Yes | No | NA |
+| BTCMarkets | Yes | No       | NA  |
+| BTSE | Yes | Yes | NA |
+| COINUT | Yes | Yes | NA |
+| Exmo | Yes | NA | NA |
+| FTX | Yes | Yes | No |
+| CoinbasePro | Yes | Yes | No|
+| Coinbene | Yes | No | No |
+| GateIO | Yes | Yes | NA |
+| Gemini | Yes | Yes | No |
+| HitBTC | Yes | Yes | No |
+| Huobi.Pro | Yes | Yes | NA |
+| ItBit | Yes | NA | No |
+| Kraken | Yes | Yes | NA |
+| Lbank | Yes | No | NA |
+| LakeBTC | Yes | No | NA |
+| LocalBitcoins | Yes | NA | NA |
+| OKCoin International | Yes | Yes | No |
+| OKEX | Yes | Yes | No |
+| Poloniex | Yes | Yes | NA |
+| Yobit | Yes | NA | NA |
+| ZB.COM | Yes | Yes | NA |
+```
+
+###### exchanges\support.go:
+```go
+var Exchanges = []string{
+	"binance",
+	"bitfinex",
+	"bitflyer",
+	"bithumb",
+	"bitmex",
+	"bitstamp",
+	"bittrex",
+	"btc markets",
+	"btse",
+	"coinbasepro",
+	"coinbene",
+	"coinut",
+	"exmo",
+	"ftx",
+	"gateio",
+	"gemini",
+	"hitbtc",
+	"huobi",
+	"itbit",
+	"kraken",
+	"lakebtc",
+	"lbank",
+	"localbitcoins",
+	"okcoin international",
+	"okex",
+	"poloniex",
+	"yobit",
+    "zb",
+```
+
+###### exchanges\exchange_test.go:
+```go
+func TestExchange_Exchanges(t *testing.T) {
+	t.Parallel()
+	x := exchangeTest.Exchanges(false)
+	y := len(x)
+	if y != 28 { // add 1 here (before FTX was added it was 27, so 28 now)
+		t.Fatalf("expected 28 received %v", y) // add 1 here
+	}
+}
+```
+
+###### cmd\documentation\exchange_templates:
+
+- Create a new file named <exchangename>.tmpl
+- Copy contents of template from another exchange example here being Exmo
+- Replace names and variables as shown:
+
+```go
+{{define "exchanges exmo" -}} // exmo -> ftx
+{{template "header" .}}
+## Exmo Exchange
+
+### Current Features
+
++ REST Support // if websocket or fix are supported, add that in too
+```
+
+```go
+main.go
+var e exchange.IBotExchange // e -> f
+
+for i := range bot.Exchanges {
+  if bot.Exchanges[i].GetName() == "Exmo" { // Exmo -> FTX
+    e = bot.Exchanges[i] // e -> f
+  }
+}
+
+// Public calls - wrapper functions
+
+// Fetches current ticker information
+tick, err := e.FetchTicker() // e -> f 
+if err != nil {
+  // Handle error
+}
+
+// Fetches current orderbook information
+ob, err := e.FetchOrderbook() // e -> f (do so for the rest of the functions too)
+if err != nil {
+  // Handle error
+}
+```
+
+- Run documentation.go to generate readme file for the exchange:
+```bash
+cd gocryptotrader\cmd\documentation
+go build
+go run documentation.exe
+```
+
+This will generate a readme file for the exchange which can be found in the new exchange's folder
+
++ 5) Create functions supported by the exchange:
```

Comment:
There seems to be a consistency issue between using numbers and bullet points, it's better to use sub headings to illustrate a point and then further sub headings/ordered/unordered lists